ocs_ci/helpers/performance_lib.py

```diff
@@ -108,7 +108,7 @@ def run_command(cmd, timeout=600, out_format="string", **kwargs):
     return output
 
 
-def run_oc_command(cmd, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE):
+def run_oc_command(cmd, namespace=config.ENV_DATA["cluster_namespace"]):
```

Comment:
Same as above, change the default value to `None` and set it to the default from config in the code also here please.